.github/workflows/ci.yaml

```diff
@@ -0,0 +1,35 @@
+name: Kyverno Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          path: kyverno-policies
+
+      - name: Clone Enterprise Kyverno
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          repository: nirmata/kyverno
+          path: kyverno
+          # The target branch of a pull request or the branch/tag of a push
+          ref: ${{ github.base_ref || github.ref_name }}
```

Comment:
The PR will be in `kyverno-policies` repo and this line is about checking out the branch in `nirmata/kyverno` repo. I don't get why is this needed. This should simply checkout the release branch in kyverno that we want to test